js/exporter/jspdf/export.js

```diff
@@ -68,11 +67,8 @@ export const Export = {
                 const styles = dataProvider.getStyles();
                 const dataRowsCount = dataProvider.getRowsCount();
 
-                if(tableWidth) {
-                    autoTableOptions.tableWidth = this.convertPixelsToPoints(tableWidth);
-                }
-
-                this.setColumnWidths(autoTableOptions, columns, keepColumnWidths);
+                const pdfColumnWidths = this.getDefaultPdfColumnWidths(autoTableOptions.tableWidth, dataProvider.getColumnsWidths());
+                this.updateColumnWidths(jsPDFDocument, autoTableOptions, pdfColumnWidths, keepColumnWidths);
```

Comment:
imho 'this.updateColumnWidths' should work without 'this'

Reply:
checked, didn't work without 'this'

Reply:
left unchanged